carbon-design-system/carbon
# fix(ProgressIndicator): fix focus style outline (#11773)

Closes #11773

This fixes the ProgressIndicator focus style to follow the [style guide](https://carbondesignsystem.com/components/progress-indicator/style/#interactive-states). 

**Default**
![Screenshot 2022-09-11 at 11 53 59](https://user-images.githubusercontent.com/26527460/193448386-f79d4dca-8c2d-4a74-8d15-2c2db1229f73.png)

**Focus when using keyboard**
![Screenshot 2022-09-11 at 11 52 52](https://user-images.githubusercontent.com/26527460/193448410-fa2283fa-4294-41e3-962d-f39128c9d484.png)

**Hover**
![Screenshot 2022-09-11 at 11 52 22](https://user-images.githubusercontent.com/26527460/193448396-18e49ee7-4cfe-4687-bdaf-7711a14ceb27.png)

**Clicked**
![Screenshot 2022-09-11 at 11 53 38](https://user-images.githubusercontent.com/26527460/193448403-0f95b073-5a76-4bde-b35d-03c3c6d98abc.png)

#### Changelog

**Changed**

- Changed the progress step button to focus the label only

**Removed**

- Removed unnecessary focus styles

#### Testing / Reviewing

- Ensure this PR's ProgressIndicator Interactive component has states as above (hover, focus when using keyboard, hover, clicked)
- Check the current way for focus style at https://react.carbondesignsystem.com/?path=/story/components-progressindicator--interactive
- Web Components has a focus style as an underline. However, Is the outlined style still the right way to focus these buttons? https://web-components.carbondesignsystem.com/?path=/story/components-progress-indicator--default
